scripts/missions/bastok/8_2_Enter_the_Talekeeper.lua

```diff
@@ -190,12 +190,18 @@ mission.sections =
                 onTrigger = function(player, npc)
                     local anyGhostsAlive = isGhostsAlive()
 
-                    if player:getMissionStatus(player:getNation()) == 2 and not anyGhostsAlive then
+                    if
+                        player:getMissionStatus(player:getNation()) == 2 and
```

Comment:
This is on me for not doing it the first time, but since you have the PR up, could you also toss in a change from `player:getNation()` to `mission.areaId`?